tgui/packages/tgui/interfaces/CrewCredits.tsx

```diff
@@ -0,0 +1,75 @@
+import { BooleanLike } from "common/react";
+import { useBackend } from "../backend";
+import { Box, Section } from "../components";
+import { Window } from "../layouts";
+
+interface CrewCreditsData {
+  darkMode: BooleanLike,
+  groups: GroupBlockProps[]
+}
+
+export const CrewCredits = (props, context) => {
+  const { data } = useBackend<CrewCreditsData>(context);
+  return (
+    <Window title="Crew Credits" width={600} height={600} theme={data["darkMode"] ? "neutral" : "paper"}>
+      <Window.Content scrollable>
+        {data.groups?.map((group, index) =>
+          (!!(data.groups[index].crew.length > 0) && <GroupBlock key={index} group={group.group} crew={group.crew} />)
+        )}
+      </Window.Content>
+    </Window>);
+};
+
+interface GroupBlockProps {
+  group: string,
+  crew: CrewMemberProps[]
+}
+
+const GroupBlock = (props:GroupBlockProps) => {
+  const { group, crew } = props;
+  const group_title = group;
```

Comment:
If you want to destructure and rename the variable for use in scope, can do as:
```suggestion
  const { group: group_title, crew } = props;
```